docs/modules/ROOT/pages/references/quality-requirements/performance/resource-quota.adoc

```diff
@@ -0,0 +1,52 @@
+// According to ISO 25010 (https://iso25000.com/index.php/en/iso-25000-standards/iso-25010)
+// Resource utilization and capacity are in the Performance category.
+= Resource limit per Namespace
+
+Source::
+Developer
+
+Stimulus::
+Deploys new workload in a Namespace
+
+Environment::
+Any APPUiO Zone
+
+Artifact::
+kubectl
+
+Response::
+Workloads are denied if they exceed the resource limit.
+Workloads are the following Kubernetes resources:
+* Pods
+* ConfigMaps
+* Secrets
+* PersistentVolumeClaims
+* Services
+* ImageStreams
+* ImageStreamTags
+
+Response measure::
+The workload is denied if they exceed one of the following Kubernetes resources:
+* CPU (all containers in all Pods)
+** Requests: 2000m
+** Limits: 4000m
+* Memory (all containers in all Pods)
+** Requests: 3 GiB
+** Limits: 6 GiB
+* Storage:
+** Request: 50 GiB (total)
+** PersistentVolumeClaims: 10
+* ConfigMaps: 150
+* Secrets: 150
+* Services: 20
+* Images
+** ImageStream Tags: 50 (per ImageStream)
+** ImageStreams: 20
```

Comment:
These numbers are certainly not set in stone. Like software, they live and breathe in the document.
But to make the requirement measurable, some numbers need to be defined.